creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -2,7 +2,10 @@ const promClient = require('prom-client')
 const _ = require('lodash')
 const config = require('../../config')
 const { exponentialBucketsRange } = require('./prometheusUtils')
-const { JOB_NAMES } = require('../stateMachineManager/stateMachineConstants')
+const {
+  JOB_NAMES,
```

Comment:
not super clear what JOB_NAMES means can we import it as `const { JOB_NAMES: STATE_MACHINE_JOB_NAMES }` or similar

Reply:
done 👍